src/fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -52,9 +52,17 @@
     ),
 )
 
-timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path)
+ophys_times = get_timestamps_from_csv(file_path=ophys_timestamp_file_path)
+behavior_times = get_timestamps_from_csv(file_path=behavior_data_file_path)
+# The timings of optical imaging are missing timezone information, therefore
+# we are adding the timezone information to the first time to get the offset
+offset = behavior_times[0] - ophys_times[0].replace(tzinfo=ZoneInfo("US/Eastern"))
```

Comment:
Can the timezone be inferred automatically from the timestamps that do have it set? (in case it ever changes in future experimenters using this setup)

Reply:
You're right, that makes more sense.